LimeBean/BeanApi.cs

```diff
@@ -573,6 +573,10 @@ public partial class BeanApi : IDisposable, IBeanCrud, IBeanFinder, IDatabaseAcc
             set { Storage.ConvertEmptyStringToNull = value; }
         }
 
+        /// <summary>
+        /// Gets or Sets whether integers are detected and extracted if in string type variables, when storing to the database.
+        /// Query paramaterisation and the database still handle storage types
+        /// </summary>
         public bool RecognizeIntegers {
```

Comment:
Well, LimeBean has been inherently anti strongly typed and performs type conversions everywhere. To me it seems more natural if the behavior is consistent regardless of fluid mode. If a developer doesn't want this conversion to occur, he can always set the corresponding flag to false.